pkg/storage/pod_test.go

```diff
@@ -255,6 +271,60 @@ var _ = Describe("Pod storage", func() {
 			},
 		}}}))
 	})
+	It("should not use start time to return metric in one cycle for long running container", func() {
+		s := NewStorage(60 * time.Second)
+		containerStart := time.Now()
+		podRef := types.NamespacedName{Name: "pod1", Namespace: "ns1"}
+
+		By("storing first batch with pod1 metrics")
+		s.Store(podMetricsBatch(podMetrics(podRef, ContainerMetricsPoint{"container1", newMetricsPoint(containerStart, containerStart.Add(120*time.Second), 1*CoreSecond, 4*MiByte)})))
+		Expect(s.Ready()).NotTo(BeTrue())
+		checkPodResponseEmpty(s, podRef)
+	})
+	It("should return correct metric for long fresh containers", func() {
```

Comment:
What does correct here means?

Reply:
Not sure how this test differs from others, what are you testing here?

Reply:
Maybe it is really duplicated with the previous use case, I will delete it